src/mam4xx/mo_setinv.hpp

```diff
@@ -0,0 +1,123 @@
+#ifndef MAM4XX_MO_SETINV_HPP
+#define MAM4XX_MO_SETINV_HPP
+
+#include <haero/atmosphere.hpp>
+
+#include <mam4xx/aero_config.hpp>
+#include <mam4xx/mam4_types.hpp>
+
+namespace mam4 {
+namespace mo_setinv {
+// number of invariants
+static const int nfs = 8;
```

Comment:
FYI, `nfs` is available as a `constexpr` in the `mam4::gas_chemistry` namespace. See `gas_chem_mechanism.hpp`.

Reply:
Ah, good catch! I gave that a shot previously, and it was crashing my builds for some reason, but that may have been related to something else--I'll swap that out!